Shared/Samples/Add ENC exchange set/AddENCExchangeSetView.swift

```diff
@@ -0,0 +1,181 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct AddENCExchangeSetView: View {
+    /// The tracking status for the loading operation.
+    @State private var isLoading = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    
+    var body: some View {
+        MapViewReader { mapProxy in
+            MapView(map: model.map)
+                .onDrawStatusChanged { drawStatus in
+                    // Updates the state when the map's draw status changes.
```

Comment:
I am not sure this is necessary. Generally, we use a `ProgressView` to indicate on-going async operations, not for rendering. I think removing this `onDrawStatusChanged` modifier and simply setting `isLoading = false` after calling `addENCExchangeSet` is sufficient. Up to you, though.

Reply:
Yeah, you're right, it doesn't need that logic anymore.